### PR TITLE
Set codecov patch status checks to `informational`

### PR DESCRIPTION
**Pull request by marijnvanderhorst**
* This _should_ make the status check not show up as a fail anymore in PRs.
* Added [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file, for instance to be used # in rulesets.
* Dependabot config that should ignore updates in `major` version of packages.
* Add compatibility with Dependabot PRs in [the Github Actions CI Workflow](https://github.com/esac-ic/esac.nl/actions/runs/12040143223).

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
         target: auto
         threshold: 25%  # Allows coverage to drop by up to 30% without failing
 
-    patch:  # Applies only to the changes in the pull request
+    patch: # Applies to the contents of a single PR or commit only.
       default:
-        target: auto
-        threshold: 25%  # Allows coverage to drop by up to 30% without failing
\ No newline at end of file
+        informational: false
```
Comment (marijnvanderhorst) [inline codecov.yml:10]:
```suggestion
        informational: true
```

**Review by ThijsEgberts — COMMENTED** [review]
I don't really know much about these configurations so I'm not sure if I should give an approving review, but from just looking at the changes it seems like it should do what you're trying to do

**Comment by marijnvanderhorst** [comment]
Yeah I understand. It's anyhow next to impossible to guarantee that all parts of this will work once merged. This, they are very hard to test from a feature branch.

The most important part is to verify that this PR does not break something else that was working before. I.e. [this CI workflow](https://github.com/esac-ic/esac.nl/actions/workflows/build.yml). 

Since the status checks that are coming from that workflow, at the bottom of this PR, both succeed and the docker image is still being pushed to the registry there should be no issues expected. At worst, something that was already broken is still broken after merging this PR

**Review by ThijsEgberts — APPROVED** [review]
@marijnvanderhorst then I'll approve

**Pull request merged**